tests/FSharp.Compiler.ComponentTests/Language/AttributeCheckingTests.fs

```diff
@@ -41,4 +41,54 @@ type C() =
         """
          |> ignoreWarnings
          |> compile
-         |> shouldSucceed
\ No newline at end of file
+         |> shouldSucceed
+
+    [<Fact>]
+    let ``ObsoleteAttribute is not taken into account when used on a type.`` () =
+        Fsx """
+open System
+
+[<Obsolete("Foo", true)>]
+type Foo() =
+  
+    member _.Bar() = ()
+        """
+        |> ignoreWarnings
```

Comment:
is ignore warnings really necessary.  Since it seems like it would obscure some classes of bugs.

Reply:
Hm, yeah, this testcase makes no sense, since it's just declaring the type with attribute and not checking the use of it.

Reply:
Yeah. I realize that this test is not useful at all . :(.  I have Fixed making sure that is now a useful one